routes/web.php

```diff
@@ -19,14 +18,19 @@
 //Route::get('/', function () {
   //  return view('welcome');
 //});
-
+Route::redirect('/', '/homepage');
 Route::get('/homepage', function(){return view('homepage');})->name('homepage');
-Route::get('/booklist', function(){return view('booklist');})->name('booklist');
-
+Route::get("/book-list",[BookController::class,'show'])->name('booklist');
+//Route::get('/book-list', function(){return view('booklist');})->name('booklist');
 
-Route::get("/adminpanel", [Admincontrol::class,'admin'])->name('adminpanel');
-Route::get("/logout",[Admincontrol::class,'logout'])->name('logout');
-Route::get("/AdminCrudInsert",[CrudOperator::class,'create'])->name('Insert');
-Route::post("/AdminCrudInsert",[CrudOperator::class,'bookcreate'])->name('BookCreate');
-Route::get("/AdminCrudUpdate",[CrudOperator::class,'update'])->name('Update');
-Route::get("/AdminCrudDelete",[CrudOperator::class,'delete'])->name('Delete');
+Route::middleware(['auth'])->group(function(){
+  Route::get("/adminpanel", [Admincontrol::class,'admin'])->name('adminpanel');
+  Route::get("/adminpanel", [BookController::class,'bookShow'])->name('adminpanel');
```

Comment:
bu Admincontrol classı ne oluyor? bu class bir controller olamaz sanırım. Benim bildiğim controller classları Pascal Case oluyor ve `{Name}Controller` conventionında oluyor.

/adminpanel sayfası tam olarak ne oluyor? buradaki isimlendirmeler sanırım pek doğru olmamış. Galiba sen kitap listeleme ve detay sayfalarını authenticated olduğundaki case için duplicate etmişsin düzenleme vs. için 

eğer öyleyese bunlar için ayrıca sayfalara gerek yok. mevcut normal blade ler içerisinde @if bloklarıyla düzenleme ekleme butonlarını koşula bağlayarak sorunu tek sayfada çözebilirsin. bu şekilde daha karmaşıklaşıyor yapılan iş. (tabi dediğim gibiyse)